README_OSX.md

```diff
@@ -0,0 +1,23 @@
+# macOS (OS X) README
+## Requirements
+As of Armory 0.95.1, due to Python requirements that can't be met by macOS 10.12 (Sierra), users must use `brew` to install an updated version of OpenSSL in order for Armory to run. Please follow these instructions.
+
+1. Open a terminal and run the `brew` command. If `brew` is not found, please execute the following commands from the command line. If the final command returns any errors, consult Google, the *bitcoin-armory* IRC channel on Freenode, or the [Bitcoin Forum](https://bitcointalk.org/index.php?board=97.0) for further instructions.
+
+        /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        touch ~/.bashrc
+        echo "export CFLAGS=\"-arch x86_64\"" >> ~/.bashrc
```

Comment:
Good question. I'm going off [directions](https://gist.github.com/bsmt/starred) that seem to have been copied & pasted throughout the Internet. I'm guessing the .bashrc lines are meant to ensure that a 32-bit version isn't built by accident.